vite.config.ts

```diff
@@ -18,13 +18,18 @@ export default defineConfig(({ mode }) => {
 
   return {
     build: {
-      chunkSizeWarningLimit: 900,
       outDir,
       rollupOptions: {
         input: {
           main: resolve(__dirname, 'index.html'),
           _scrivito_extensions: resolve(__dirname, '_scrivito_extensions.html'),
         },
+        output: {
+          manualChunks: {
+            react: ['react-bootstrap', 'react-helmet-async', 'react-toastify'],
```

Comment:
`'react-dropzone'` is also a react dependency, that is used in `DataFormUploadWidget`. Please add it to the list.

Reply:
`react-dropzone` is only used in editing. I'd like to avoid having editing-only dependencies in manual chunks.

Reply:
No, this package is used for the regular `DataFormUploadWidget`:

https://github.com/Scrivito/scrivito-portal-app/blob/e073bfb082f2fa25c9c49a16d2b1bb436fbf9b81/src/Widgets/DataFormUploadWidget/DataFormUploadWidgetComponent.tsx#L26-L30

See it in action on https://main.scrivito-portal-app.pages.dev/en/create-ticket-f909e9691db27360.

Reply:
True, I wrongly assumed dropzone goes together with draggable. 😄